service-mesh-probe/service-mesh.proto

```diff
@@ -67,11 +67,11 @@ enum Protocol {
     TCP = 2;
 }
 
-message Throughput {
+message Tcp {
```

Comment:
Should be all upper case, I think.

Reply:
Renamed to `TCPInfo` because `TCP` is defined in `enum Protocol`